src/go/k8s/pkg/resources/secret.go

```diff
@@ -194,6 +194,11 @@ preStopHook () {
 	echo "Setting maintenance mode on node ${NODE_ID}"
 	until [ "${status:-}" = "200" ]; do
 		status=$(%s)
+    if [ "${status:-}" = "404" ]; then
```

Comment:
nit: please indent this block to match the indent of the rest of this script.

Reply:
was a case of tabs vs spaces.. fixed